gtest/src/runner.rs

```diff
@@ -59,20 +59,34 @@ impl CollectState for Storage<InMemoryMessageQueue, InMemoryProgramStorage, InMe
 impl CollectState for Storage<ExtMessageQueue, ExtProgramStorage, ExtWaitList> {
     fn collect(self) -> FinalState {
         let log = self.message_queue.log;
+        let program_storage = self.program_storage;
 
         let mut messages = Vec::new();
-
-        let mut message_queue =
-            common::storage_queue::StorageQueue::get("g::msg::".as_bytes().to_vec());
+        let mut message_queue = common::storage_queue::StorageQueue::get(b"g::msg::".as_ref());
         while let Some(message) = message_queue.dequeue() {
             messages.push(message);
         }
 
+        let program_ids: HashSet<ProgramId> =
```

Comment:
This is interesting way to collect all programs, but it should have been a comment that we can assert page status of programs that had at least one outgoing/incoming message

Reply:
And a TODO: + issue to fix it later in comment

Reply:
Collecting programs reworked in new commit